.gitattributes

```diff
@@ -0,0 +1,8 @@
+.distignore export-ignore
+.github export-ignore
+.gitattributes export-ignore
+.gitignore export-ignore
+composer.lock export-ignore
+docker-compose.yml export-ignore
+package-lock.lock export-ignore
+phpcs.xml export-ignore
```

Comment:
This ensures all these files won't be included when installing this package with composer, as it uses git under the hood.